package.json

```diff
@@ -7,128 +7,128 @@
+    "eslint-config-standard-react": "~6.0.0",
+    "eslint-plugin-import": "~2.10.0",
+    "eslint-plugin-jest": "~21.15.0",
+    "eslint-plugin-node": "~6.0.1",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
-    "eslint-plugin-promise": "~3.6.0",
+    "eslint-plugin-promise": "~3.7.0",
     "eslint-plugin-react": "~7.7.0",
     "eslint-plugin-standard": "~3.0.1",
-    "file-loader": "~1.1.5",
-    "flow-bin": "^0.66.0",
+    "file-loader": "~1.1.11",
+    "flow-bin": "^0.69.0",
     "front-matter": "~2.3.0",
     "generate-package-json-webpack-plugin": "^1.0.0",
-    "git-repo-version": "~1.0.1",
+    "git-repo-version": "~1.0.2",
     "highlight.js": "~9.12.0",
-    "html-loader": "~0.5.1",
+    "html-loader": "~0.5.5",
     "identity-obj-proxy": "~3.0.0",
     "img-loader": "~2.0.1",
-    "is-url": "^1.2.2",
-    "jest": "~22.4.2",
-    "jest-fetch-mock": "~1.4.0",
+    "is-url": "^1.2.4",
+    "jest": "~22.4.3",
+    "jest-fetch-mock": "~1.5.0",
     "json-loader": "~0.5.7",
-    "lolex": "~2.3.1",
-    "path-to-regexp": "~2.1.0",
+    "lolex": "~2.3.2",
+    "path-to-regexp": "~2.2.0",
     "pixrem": "~4.0.1",
     "pleeease-filters": "~4.0.0",
-    "postcss": "~6.0.14",
+    "postcss": "~6.0.21",
     "postcss-calc": "~6.0.1",
     "postcss-color-function": "~4.0.1",
     "postcss-custom-media": "~6.0.0",
-    "postcss-custom-properties": "~6.2.0",
+    "postcss-custom-properties": "~7.0.0",
     "postcss-custom-selectors": "~4.0.1",
-    "postcss-flexbugs-fixes": "~3.2.0",
-    "postcss-import": "~11.0.0",
-    "postcss-loader": "~2.0.9",
+    "postcss-flexbugs-fixes": "~3.3.0",
+    "postcss-import": "~11.1.0",
+    "postcss-loader": "~2.1.3",
     "postcss-media-minmax": "~3.0.0",
     "postcss-mixins": "~6.2.0",
     "postcss-nesting": "~4.2.1",
     "postcss-selector-matches": "~3.0.1",
     "postcss-selector-not": "~3.0.1",
-    "prop-types": "~15.6.0",
+    "prop-types": "~15.6.1",
     "raf": "~3.4.0",
-    "react-hot-loader": "~4.0.0",
-    "react-scroll": "~1.7.5",
-    "redux-mock-store": "~1.3.0",
+    "react-hot-loader": "~4.0.1",
+    "react-scroll": "~1.7.8",
+    "redux-mock-store": "~1.5.1",
     "regenerator-runtime": "^0.11.1",
     "rimraf": "~2.6.2",
     "s3": "~4.4.0",
-    "style-loader": "~0.19.0",
+    "style-loader": "~0.20.3",
     "stylefmt": "~6.0.0",
-    "stylelint": "~8.3.1",
+    "stylelint": "~9.2.0",
     "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-order": "~0.8.0",
-    "stylelint-processor-styled-components": "^1.3.0",
-    "stylelint-webpack-plugin": "~0.9.0",
-    "url-loader": "~0.6.2",
-    "webpack": "~3.10.0",
-    "webpack-bundle-analyzer": "~2.9.1",
-    "webpack-dev-middleware": "~1.12.2",
-    "webpack-hot-middleware": "~2.21.0",
-    "webpack-node-externals": "^1.6.0"
+    "stylelint-order": "~0.8.1",
+    "stylelint-processor-styled-components": "^1.3.1",
+    "stylelint-webpack-plugin": "~0.10.4",
+    "url-loader": "~1.0.1",
+    "webpack": "~4.5.0",
```

Comment:
Are there any webpack plugins which are not updated yet? Like "assets-webpack-plugin"

Reply:
Woops, it seems assets-webpack-plugin didn't upgrade yet... Also it's last commit was Jan 20, 2017.
What do we need that plugin for ? 🙈

Reply:
@maxammann I suggest to merge it anyways now....